data/human/boarding missions.txt

```diff
@@ -466,7 +466,7 @@ mission "Slave Crew Rescue (No Brig Variant)"
 				`	(Get to work repairing the ship.)`
 					accept
 				`	(Press the matter.)`
-			`	Something about captain Tarquin's explanation doesn't sit right with you, so you keep asking questions to distract him while subtly walking towards the ship's crew section. As you get closer, you start to hear the faint sounds of people banging on walls and yelling for help. You turn back to Tarquin, whose face is displaying an almost uncannily neutral expression, but his body is taut as as though every muscle has tensed up all at once.`
+			`	Something about captain Tarquin's explanation doesn't sit right with you, so you keep asking questions to distract him while subtly walking towards the ship's crew section. As you get closer, you start to hear the faint sounds of people banging on walls and yelling for help. You turn back to Tarquin, whose face is displaying an almost uncannily neutral expression, but his body is as taut as though every muscle has tensed up all at once.`
```

Comment:
These two still aren't right.
As taut as _what_? I think the better change is to remove the first "as."

(This block in general has really long sentences that could probably due with editing.)

Reply:
```suggestion
			`	Something about captain Tarquin's explanation doesn't sit right with you, so you keep asking questions to distract him while subtly walking towards the ship's crew section. As you get closer, you start to hear the faint sounds of people banging on walls and yelling for help. You turn back to Tarquin, whose face is displaying an almost uncannily neutral expression, but his body is taut, as though every muscle has tensed up all at once.`
```
Here's the solution I'd suggest.
```suggestion
			`	Something about captain Tarquin's explanation doesn't sit right with you, so you keep asking questions to distract him while subtly walking towards the ship's crew section. As you get closer, you start to hear the faint sounds of people banging on walls and yelling for help. Tarquin's face displays an almost uncannily neutral expression, but his body is taut, as though every muscle has tensed up all at once.`
```
Or this for a little reduced wordiness. Thoughts?
Whatever solution we go with will also need to be implemented on line 635, of course.

Reply:
I'll go with the second option in @W1zrad's comment for now. If something else would be better, I'm happy to change it.